pkg/kubelet/volume_host.go

```diff
@@ -125,6 +125,14 @@ func (kvh *kubeletVolumeHost) GetHostIP() (net.IP, error) {
 	return kvh.kubelet.GetHostIP()
 }
 
+func (kvh *kubeletVolumeHost) GetNodeCapacity() (api.ResourceList, error) {
```

Comment:
can you leave a // TODO that says to fetch node capacity from a common `NodeCapacityProvider` interface?

we have a similar problem per pod and per qos cgroups, and we need to pass an interface around that retrieves node capacity in a common manner rather than always falling back to getNodeAnyway().

I also wonder in the future if we should modify this to be node.Status.Allocatable when per cgroup enforcement is enabled.